setup.cfg

```diff
@@ -51,3 +49,9 @@ dev =
     orquestra-python-dev
     # Used in tests
     numpy
+    pyscf~=2.2.0
```

Comment:
I think it would be actually better to do `benchq[pyscf]` instead of pyscf here, as we did [here](https://github.com/zapatacomputing/orquestra-opt/blob/4f3e5610df828e72383a43d700d2a0d256a3931b/setup.cfg#L53) .
WDYT?

Reply:
Good point, fixed in 729af580c8fccafa4385258c4503cbeb4a832be8